flycheck.el

```diff
@@ -12255,7 +12255,7 @@ information about tflint."
 See URL `https://github.com/terraform-linters/tflint'."
   :command ("tflint" "--format=json"
             (option-list "--var-file=" flycheck-tflint-variable-files concat)
-            source-original)
+            (option-list "--chdir=" source-directory concat))
```

Comment:
```suggestion
            (option-list "--filter=" buffer-file-name concat))
```

Reply:
I'd rather leave the `--chdir` option in place. At worst, it does nothing.